rainmaker_orchestrator/server.py

```
@@ -1,426 +1,628 @@
+# Application Entry Point
+# ============================================================================
+
 if __name__ == "__main__":
-    logger.info("Starting development server (not for production!)")
-    logger.warning("Use Gunicorn for production: gunicorn --bind 0.0.0.0:8080 server:app")
-    app.run(
+    import uvicorn
+
+    uvicorn.run(
+        "server:app",
         host="0.0.0.0",
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)